app/services/authenticate_installation.rb

```diff
@@ -0,0 +1,35 @@
+# Instantiate an Octokit client, authenticated as an installation of a
+# GitHub App, to run API operations.
+module AuthenticateInstallation
```

Comment:
It looks like the only place you're using this module is in your rake task. I would have combined this logic with your `AuthenticatedGithubClient` singleton so that that singleton just returns a fully working Octokit client (with the installation token set). Then in your rake task you just call something like `AuthenticatedGithubClient.instance.create_issue`